### DISCUSSION TITLE
Performance?

### DISCUSSION THREAD

**Discussion by jsomedon** [janet-lang/janet#1239] [Q&A] [answered]
How performant janet is? Say, comparing with lua,luajit,python,javascript(v8)? I think if janet's performance is kinda on par with luajit, then people would seriously consider this as a scripting language for game project.

**Comment by primo-ppcg** [comment, marked as answer]
Janet is [not marketed as a fast language](https://github.com/janet-lang/janet#how-fast-is-it), but I do think the author is under-selling it somewhat.

In my experience Janet is faster than Python, but not quite as fast as PyPy. I suspect the same is probably true for Lua / LuaJIT.

**Comment by sogaiu** [comment]
FWIW, @MikeBeller put [this](https://github.com/MikeBeller/janet-benchmarksgame) together some time back.  Not sure what the status is these days.

**Reply by pepe** [reply]
I would like to know what has changed after the year the repository was touched, especially with some of the @primo-ppcg's optimizations. BTW thanks a lot for them; I am impressed by the volume of thought you are paying to the details of the inner workings of Janet. I wish I could be that good too :-).

**Reply by primo-ppcg** [reply]
I appreciate the compliment!

> I would like to know what has changed after the year the repository was touched

I'd be interested to see that too. Perhaps I'll make an io page for it, comparing Janet 1.26 - 1.30 against Python and Lua.
